File: app/src/main/java/volovyk/guerrillamail/data/remote/pojo/CheckForNewEmailsResponse.java
package volovyk.guerrillamail.data.remote.pojo;

import com.google.gson.annotations.SerializedName;

import java.util.List;

import volovyk.guerrillamail.data.model.Email;

public class CheckForNewEmailsResponse {
    @SerializedName("list")
    private List<Email> emails;
    @SerializedName("sid_token")
    private String sidToken;

    public List<Email> getEmails() {
        return emails;
    }

    public String getSidToken() {
        return sidToken;
    }
}

File: app/src/main/java/volovyk/guerrillamail/data/remote/pojo/CheckForNewEmailsResponse.kt
package volovyk.guerrillamail.data.remote.pojo

import com.google.gson.annotations.SerializedName
import volovyk.guerrillamail.data.model.Email

class CheckForNewEmailsResponse {
    @SerializedName("list")
    val emails: List<Email>? = null

    @SerializedName("sid_token")
    val sidToken: String? = null
}